xml/html_title.xml

```diff
@@ -708,9 +748,10 @@
     <param pos="1" name="hw.product"/>
   </fingerprint>
 
-  <fingerprint pattern="^Netgear Prosafe Plus Switch$">
+  <fingerprint pattern="^(?i)Netgear Prosafe Plus Switch$">
```

Comment:
For future PRs please move inline regex flags/modifiers such as `(?i)` and `(?m)` to the beginning of the regex (prior to the anchor). In Python these flags have global meaning and use of them anywhere other than the start of the pattern is deprecated. This does not change the function of these flags in current patterns for other languages.

https://bugs.python.org/issue22493

It is quite possible that we will be making this change globally.

Reply:
Thanks for the detail there, @tsellers-r7. Will do going forward here. :+1: